index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <div class="header">
+      <div class="row">
+        <div class="logo">
+          <img src="img/youtube_logo.png"> </img>
+        </div>
+
+        <div class="search-bar">
+          <input type="text" placeholder="searching here"></input>
+          <button>S</button>
+          <div class="signIn">
+              <img src="img/uploadButton.png"> </img>
+              <img src="img/moreButton.png"> </img>
+              <img src="img/avatarButton.png"> </img>
+            </div>
+        
+        </div>
+
+        
+      </div>
+    </div>
+
+
+    <div class="row">
+
+      <div class="column1">
+
+        <div class="videoContainer">
+
+          <div class="video">
+            <div class="videoPrincipal">
+              <iframe src="https://www.youtube.com/embed/ZnuwB35GYMY"></iframe>
+            </div>
+
+            <div class="videoTitle">
+              <h2>Contributing to Open Source Part I: The Easy Way</h2>
+              
+            </div>
+
+            <div class="subs">
+              <img src="img/subscribe.png">
```

Comment:
This should be a` <button>`  instead of and image.
You should also have the avatar image separate.